x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/rules/api/util/audit.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { AuditEvent, EcsEvent } from '@kbn/core/server';
+import type { ArrayElement } from '@kbn/utility-types';
+
+export enum SiemMigrationsAuditActions {
+  SIEM_MIGRATION_STARTED = 'siem_migration_started',
+  SIEM_MIGRATION_STOPPED = 'siem_migration_stopped',
+  SIEM_MIGRATION_CREATED = 'siem_migration_created',
+  SIEM_MIGRATION_UPDATED = 'siem_migration_updated',
+  SIEM_MIGRATION_CREATED_LOOKUP = 'siem_migration_created_lookup',
+  SIEM_MIGRATION_INSTALLED_RULE = 'siem_migration_installed_rule',
+  SIEM_MIGRATION_UPDATED_RULE = 'siem_migration_updated_rule',
+}
+
+export enum AUDIT_TYPE {
+  CHANGE = 'change',
+  START = 'start',
+  END = 'end',
+  ACCESS = 'access',
+  CREATION = 'creation',
+}
+
+export enum AUDIT_CATEGORY {
+  API = 'api',
+  AUTHENTICATION = 'authentication',
+  DATABASE = 'database',
+  WEB = 'web',
+}
+
+export enum AUDIT_OUTCOME {
+  FAILURE = 'failure',
+  SUCCESS = 'success',
+  UNKNOWN = 'unknown',
+}
+
+export const siemMigrationAuditEventType: Record<
+  SiemMigrationsAuditActions,
+  ArrayElement<EcsEvent['type']>
+> = {
+  siem_migration_started: AUDIT_TYPE.START,
+  siem_migration_stopped: AUDIT_TYPE.END,
+  siem_migration_created: AUDIT_TYPE.CREATION,
+  siem_migration_updated: AUDIT_TYPE.CHANGE,
+  siem_migration_installed_rule: AUDIT_TYPE.CREATION,
+  siem_migration_updated_rule: AUDIT_TYPE.CHANGE,
+  siem_migration_created_lookup: AUDIT_TYPE.CREATION,
+};
+
+export const siemMigrationAuditEventMessage: Record<SiemMigrationsAuditActions, string> = {
```

Comment:
Unsure if there is any wishes to the text we should show in the logs, feel free to let me know